config/core/test-configs.yaml

```diff
@@ -43,6 +43,7 @@ file_systems:
     type: debian
     ramdisk: 'buster/20210913.0/{arch}/initrd.cpio.gz'
     nfs: 'buster/20210913.0/{arch}/full.rootfs.tar.xz'
+    compression: gz
```

Comment:
The problem with this is there is a different compression for each rootfs image (e.g. nfs and ramdisk).  So I think relying on the file name extension works better in practice.

To support a `compression` attribute, we would need to have a configuration object for each rootfs image.  At the moment we just have a string (URI).

Reply:
actually we don't if we set a default compression attribute.

Reply:
And we can't do it at the moment, unless we add a configuration object (e.g. YAML dictionary) for each rootfs image as they may all have different compression like in this particular example (ramdisk is gz, nfsroot is xz).

Reply:
My Idea was to have a default compression attribute xz if no attribute is specified. So that the compression attribute can be added only for compression usage different from xz and not changing the current test-configs.yaml template

Reply:
The compression attribute is defined in the template:
https://github.com/kernelci/kernelci-core/blob/main/config/lava/base/kernel-ci-base-tftp-deploy.jinja2#L24

Reply:
ok. so what I have to do ?